src/controllers/TotpController.php

```diff
@@ -40,6 +46,29 @@ public function behaviors()
                         'roles' => ['@'],
                         'allow' => true,
                     ],
+                    [
+                        'actions' => ['api-temporary-secret', 'api-disable', 'api-enable'],
```

Comment:
```suggestion
                        'actions' => ['api-temporary-secret', 'api-disable', 'api-enable'],
                        'roles' => ['@'],
```

Эти методы должны быть доступны только аутентифицированным, нет?